hera_sim/tests/test_vis.py

```diff
@@ -312,7 +313,7 @@ def align_src_to_healpix(point_source_pos, point_source_flux, nside=2 ** 4):
     )
 
     hmap[:, pix] += point_source_flux / healpy.nside2pixarea(nside)
-    nside = healpy.get_nside(hmap[0])
+    nside = healpy.npix2nside(len(hmap[0]))
```

Comment:
This can also be accomplished without the `healpy` part of `astropy-healpix` (which is less preferred) using `astropy_healpix.npix_to_nside`.